sdks/python/apache_beam/transforms/ptransform_test.py

```diff
@@ -550,11 +563,23 @@ def test_flatten_pcollections_in_iterable(self):
     assert_that(result, equal_to([0, 1, 2, 3, 4, 5, 6, 7]))
     pipeline.run()
 
+  @attr('ValidatesRunner')
+  def test_flatten_a_flattened_pcollection(self):
+    pipeline = TestPipeline()
+    pcoll_1 = pipeline | 'Start 1' >> beam.Create([0, 1, 2, 3])
+    pcoll_2 = pipeline | 'Start 2' >> beam.Create([4, 5, 6, 7])
+    result = ((pcoll_1, pcoll_2) | 'Flatten' >> beam.Flatten(),
```

Comment:
This is a bit hard to read. 

It also would probably be good to have a third collection in the mix, e.g. 

```
pcoll_12 = (pcoll_1, pcoll_2) | beam.Flatten()
pcoll_123 = (pcoll_12, pcoll_3) | 'FlattenAgain' >> beam.Flatten()
```

Reply:
Refactored as suggested:)